_includes/v21.2/sql/shell-options.md

```diff
@@ -7,8 +7,9 @@
 Client Options | Description
 ---------------|------------
 <a name="sql-option-auto-trace"></a> `auto_trace` | For every statement executed, the shell also produces the trace for that statement in a separate result below. A trace is also produced in case the statement produces a SQL error.<br><br>**Default:** `off`<br><br>To enable this option, run `\set auto_trace on`.
+<a name="sql-option-border"></a> `border` | Display a border around the output of the SQL statement when using the `table` display format. Set the level of borders using `border=<level>` to configure how many borders and lines are in the output, where `<level>` is an integer between 0 and 3. The higher the integer, the more borders and lines are in the output. <br>A level of `0` shows the output with no outer lines and no row line separators. <br>A level of `1` adds row line separators. A level of `2` adds an outside border and no row line separators. A level of `3` adds both an outside border and row line separators. <br><br>**Default:** `0` <br><br>To change this option, run `\set border=<level>`. [See an example](cockroach-sql.html#show-borders-around-the-statement-output-within-the-sql-shell).
```

Comment:
Does this need `<span class="version-tag">New in v21.2</span>`?

Reply:
Done.